tasks/models.py

```diff
@@ -68,6 +68,9 @@ def change_assignee(self, new_assignee):
             raise ValueError("The new assignee must be of the same team")
         self.assignee = new_assignee
 
+    def get_comments(self):
+        return Comment.objects.filter(task=self).order_by('id')
+
```

Comment:
where do you import Comment in this file?

Reply:
Comment model is defined in the same file